pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -1732,6 +1732,19 @@ func forceTableGC(
 	}
 }
 
+func forceTableGCAtTimestamp(
+	t testing.TB,
+	tsi serverutils.TestServerInterface,
+	sqlDB *sqlutils.SQLRunner,
```

Comment:
nit: unused param

(might as well delete the analogous unused param in `forceTableGC` while you're at it)